pkgs/applications/emulators/retroarch/wrapper.nix

```
@@ -1,13 +1,30 @@
 { lib
 , stdenv
+, cores ? [ ]
+, extraPaths ? [ ]
+, withAssets ? true
+, withCoreInfo ? true
+, libretro-core-info
 , makeWrapper
 , retroarch
+, retroarch-assets
 , symlinkJoin
 , writeTextDir
-, cores ? [ ]
 }:
 
 let
+  retroarch' = retroarch.overrideAttrs (old: {
+    patches = (old.patches or [ ]) ++ [
```

```suggestion
    patches = old.patches or [ ] ++ [
```
you don't need them here